src/Microsoft.SqlTools.ServiceLayer/LanguageServices/PeekDefinition.cs

```diff
@@ -149,6 +182,25 @@ internal Location[] GetScript(IEnumerable<Declaration> declarationItems, string
         }
 
         /// <summary>
+        /// Return schema name from the full name of the database. If schema is missing return dbo as schema name.
+        /// </summary>
+        /// <param name="fullObjectName"> The full database qualified name(database.schema.object)</param>
+        /// <param name="objectName"> Object name</param>
+        /// <returns>Schema name</returns>
+        internal string GetSchemaFromDatabaseQualifiedName(string fullObjectName, string objectName)
+        {
+            string[] tokens = fullObjectName.Split('.');
+            for (int i = tokens.Length - 1; i >= 0; i--)
```

Comment:
>>= [](start = 46, length = 2)

Why go through the loop for i = 0? You know (and explicitly check) that it shouldn't do anything